tests/unit/core/reducers/test_versions.js

```diff
@@ -440,6 +440,42 @@ describe(__filename, () => {
         ).toEqual(createInternalVersion(version));
       });
 
+      it('maintains licence and releaseNotes from pre-existing versions', () => {
```

Comment:
s/licence/license/

Reply:
nit: also can releaseNotes just be release notes (or release_notes if you're trying to match the var name)?